gateway/src/run.rs

```diff
@@ -100,11 +100,11 @@ pub fn execute(
 
     // Start RPC servers.
     info!(logger, "Starting WS server"; "conf" => ?ws_conf);
-    let ws_server = rpc::new_ws(ws_conf, &dependencies).map_err(|err| format_err!("{}", err))?;
+    let ws_server = rpc::new_ws(ws_conf, &dependencies).map_err(|err| anyhow!("{}", err))?;
```

Comment:
```suggestion
    let ws_server = rpc::new_ws(ws_conf, &dependencies).map_err(Error::msg)?;
```
[Error::msg](https://docs.rs/anyhow/1.0.31/anyhow/struct.Error.html#method.msg)